src/app/ToDo/state/todo.httpservice.ts

```diff
@@ -1,21 +1,16 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { from, Observable } from 'rxjs';
-import ToDo from './todo.model';
+import { ToDo } from '.';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root'
 })
 export class ToDoHttpService {
```

Comment:
We don't use the `Http Service` and` Service` divisions, feel free to rename the class to `TodoService`.